app/lib/db.js

```diff
@@ -6,6 +6,8 @@ const mainClient = redis.createClient(
     config.redis.main.host,
     {
         auth_pass: config.redis.main.password,
+        prefix: config.redis.main.prefix || undefined,
```

Comment:
Do the `undefined` defaults work with a redis setup that doesn't use these? I want to make sure that folks happily using the current config will not need to make changes. 

Why not use `null` and `false` as defaults?

Reply:
Good catch, I'm not sure why I used undefined. I suppose the undefined is not necessary - if a user doesn't provide anything it would be undefined (as it is currently) so I can just remove the || undefined and it would work for folks who don't use this.

Reply:
I'm testing this fix - will resolve when I've verified it

Reply:
Okay - I've squashed and pushed everything. I did test two scenarios:

1. default with no tls/no prefix enabled on a localhost
2. remote aws tls enabled redis server cluster - with tls and prefix passed. 

Both scenarios worked out for me. Ready for another review whenever there's some time to :)

Reply:
Thanks so much for giving it another look and making sure it works in both scenarios! We're wrapping up a few things before preparing a release and will try to get a final review and merge in before then.

If you ever customize widgets, you may want to look at https://github.com/enketo/enketo-express/issues/495#issuecomment-1499621829

Reply:
Sounds good! Yeah, no rush, its better to have a full review than release a bug :+1: